rados/conn.go

```diff
@@ -143,22 +145,34 @@ func (c *Conn) SetConfigOption(option, value string) error {
 	return getRadosError(int(ret))
 }
 
+var (
+	getConfigOptionStartSize = 4096    // 4KiB
```

Comment:
This buffer looks pretty large for a configuration option value. Of course this was the earlier default, but starting with 128 bytes would probably be sufficient for many options (booleans, integers, names, ...).

Reply:
I agree. But I'd rather handle that in a separate PR. In fact I'll probably undo moving it to a variable outside of the function next time I rebase. I will file an issue to track updating some of those initial sizes.